### DISCUSSION TITLE
How do I style the active menu item when their is a sub-menu active?

### DISCUSSION THREAD

**Discussion by littlefengers** [ariakit/ariakit#837] [Questions] [answered]
Hi

In the Submenu example, the active menu item when you go into a submenu has a blue background, how do I do that? Is that just a CSS selector? Do I have to use a currentId?

Thank You!

**Comment by diegohaz** [comment, marked as answer]
Yeah, that's just a css selector (`[aria-expanded="true"]`).

**Reply by drixie** [reply]
@diegohaz How does one use this to dynamically set blue background style on the active menu item?
